exec/java-exec/pom.xml

```diff
@@ -889,11 +914,11 @@
               <lookAhead>2</lookAhead>
               <isStatic>false</isStatic>
               <outputDirectory>${project.build.directory}/generated-sources/</outputDirectory>
-<!--
```

Comment:
Do you know why we have this commented out code in this `pom.xml` file?   I'd imagine it is here for a reason, and if so, we could indicate that.   If you have no idea, it's fine and we can just leave it.